docs/Migration-2.0.md

```diff
@@ -0,0 +1,45 @@
+# Migration from 1.12.x to 2.0.x
+
+This guide provides instructions for migrating from Wultra Mobile Token SDK for iOS version `1.12.x` to version `2.0.x`.
+
+Version `2.0.x` introduces a significant simplification of the SDK’s design and usage.
+
+---
+
+### Added Functionality
+**Preferred Instantiation Method**  
+   The `WultraMobileToken` class is now the recommended way to instantiate the SDK.
+   PowerAuthSDK extension method is provided for this purpose.
+   
+```kotlin
+func createWultraTokenMobile(powerAuth: PowerAuthSDK) {
+    do {
+        let wmt = powerAuth.createWultraMobileToken()
+    
+        let operationsService = wmt.operations
+        let pushService = wmt.inbox
+        let inboxService = wmt.push
+    
+    } catch IntiError.invalidBaseURL(let url) {
+        // PowerAuth baseUrl is not valid
+    }
+}
+```
+
```

Comment:
It would be nice to add "WMTOperations, ...` are now concrete classes instead of protocols.